msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties

```diff
@@ -100,6 +100,8 @@ stat_sort_by_date=Sort by Date
 stat_byUser=by User
 stat_byTopic=by Topic
 stat_totalMessages=Total Messages
+stat_percent_read=<span class="statsTally">Percent Read:</span> {0}%
+stat_num_authored=<span class="statsTally">Messages Authored:</span> {0}
```

Comment:
These `<span>`'s are probably better to have in the JSP. Let me know if I should refactor.